getpfp.py

```diff
@@ -0,0 +1,75 @@
+# -*- coding: future_fstrings -*-
+
+#    Friendly Telegram (telegram userbot)
+#    Copyright (C) 2018-2019 The Authors
+
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as published by
+#    the Free Software Foundation, either version 3 of the License, or
+#    (at your option) any later version.
+
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import logging
+
+from .. import loader, utils
+
+logger = logging.getLogger(__name__)
+
+
+def register(cb):
+    cb(GetPPMod())
```

Comment:
Can you change that to GetPFPMod? I feel like someone left a joke in production.